src/components/Alert/AlertBanner.vue

```diff
@@ -0,0 +1,156 @@
+      }]">
+        <div v-show="text" class="banner-alert__container-text">
+          <unnnic-icon class="banner-alert__textIcon" v-show="isShowTextIcon(type)" :icon="getIconType(type)" size="sm" scheme="neutral-white" />
+          <span class="text" >{{ text }}</span>
+          <a v-if="linkHref" class="banner-alert__link" :href="linkHref" :target="linkTarget">
+            {{ linkText }}
+          </a>
+        </div>
+  
+        <div
+          v-show="showCloseButton"
+          class="banner-alert__close"
+          @click="emitClose"
+        >
+          <unnnic-icon icon="close" size="sm" scheme="neutral-white" />
+        </div>
+      </div>
+  </template>
+  
+  <script>
+  import UnnnicIcon from '../Icon.vue';
+  
+  export default {
+    components: {
+      UnnnicIcon,
+    },
+    methods: {
+      getIconType(type) {
+        if (type === 'danger') return 'block';
+        if (!type.trim()) return 'info'; //check if type is empty or whitespace
+        return type;
+      },
+      isShowTextIcon(type) {
+        return ['danger', 'warning', 'info', ''].includes(type)
+      },
+      emitClose() {
+        this.onClose();
+
+        this.$emit('close');
+      }
+    },
+    props: {
+      text: {
+        type: String,
+        default: null,
+      }, 
+      onClose: {
+        type: Function,
+        default: () => {}
+      },
+      showCloseButton: {
+        type: Boolean,
+        default: false
+      },
+      linkHref: {
+        type: String,
+        default: '',
+      },
+      linkTarget: {
+        type: String,
+        default: '_blank',
+      },
+      linkText: {
+        type: String,
+        default: 'Learn more',
+      },
+      type: {
+        type: String,
+        default: 'info',
+      },
+    }
+  };
+  </script>
+  
+  <style lang="scss" scoped>
+  @import '../../assets/scss/unnnic.scss';
+  
+  .banner-alert {
+    display: flex;
+    align-items: center;
+    text-align: center;
+    justify-content: space-between;
+    margin-bottom: $unnnic-spacing-sm;
+  
+    color: $unnnic-color-neutral-white;
+    font-family: $unnnic-font-family-secondary;
+    font-size: $unnnic-font-size-body-gt;
+    line-height: $unnnic-font-size-body-gt + $unnnic-line-height-md;
+    font-weight: $unnnic-font-weight-regular;
+    background-color: $unnnic-color-aux-blue-500;
+
+    &__container-text {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: $unnnic-spacing-xs $unnnic-spacing-sm;
+      flex-grow: 1;
+
+      .text {
+        margin-right: $unnnic-spacing-nano;
+      }
+    }
+
+    &__textIcon {
+      margin-right: $unnnic-spacing-xs;
+    }
+  
+    &__link {
+      color: inherit;
+      display: block;
+      font-weight: $unnnic-font-weight-bold;
+      text-decoration: underline;
+      text-underline-position: under;
+    }
+  
+    &__close {
+      display: flex;
+      padding: ($unnnic-spacing-xs*1.375) $unnnic-spacing-sm;
```

Comment:
Why "1,375"? If the design predicts 11px, it's probably wrong

Reply:
In this situation, can I use the static value then?

Reply:
Or do I create a new space token?